.travis.yml

```diff
@@ -81,6 +81,7 @@ env:
      - "TESTDIR=Java/wildfly-ee7"
      - "TESTDIR=Java/wizzardo-http"
      - "TESTDIR=Scala/cask"
+     - "TESTDIR=JavaScript/graphql-express-mysql"
```

Comment:
Remove this please, the line below will ensure it gets tested with the other JavaScript impls.